application.js

```diff
@@ -46,31 +48,85 @@ var sessionOptions = {
   }
 };
 
+/**
+ * Gets mongodb connection information from $fh.db
+ * Use Data Browser > Upgrade Database to get the connection information available
+ */
+function mongoInit(cb) {
+  $fh.db({
+    act: 'connectionString'
+  }, function(err, connectionString) {
+    if (err) {
+      return cb(err);
+    }
+    if (!connectionString) {
```

Comment:
Changed the logging to `error` and with extra advice for upgrading the database.

Running without finding the connection string is still valid for local dev, unless we always set `FH_MONGODB_CONN_URL=mongodb://localhost` every time instead.